hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/BlockID.java

```diff
@@ -79,21 +104,33 @@ public String toString() {
   public void appendTo(StringBuilder sb) {
     containerBlockID.appendTo(sb);
     sb.append(" bcsId: ").append(blockCommitSequenceId);
+    sb.append(" replicaIndex: ").append(replicaIndex);
+    sb.append(Arrays.stream(stackTraceElements).map(StackTraceElement::toString).collect(Collectors.joining("\n")));
```

Comment:
Do we really want the stack trace to be printed any thing something calls toString() on a blockID object?

Reply:
I forgot to remove this. I had added it to debug my test case. I will remove it.

Reply:
Removed it.